gear-test/spec/test_failed_program_msg_handling.yaml

```
@@ -21,11 +21,18 @@ fixtures:
           value: 100
         payload:
           kind: utf-8
-          value: unknown
+          value: unknownspec_version correction v2
```

Who is this message for? :D 

```suggestion
          value: unknown
```

---

done